### PR TITLE
Allow cluster size parameters to be set by caller.

### PR DESCRIPTION
**Pull request by tcnghia**
This would allow some PR/repository to change cluster size without changing it for everyone else.

### CODE REVIEW

File: scripts/e2e-tests.sh [hunk 1]
```diff
@@ -147,11 +151,12 @@ function create_test_cluster() {
   set -o errexit
   set -o pipefail
 
-  header "Creating test cluster"
+  header "Creating test cluster with --min-nodes=${E2E_MIN_CLUSTER_NODES} --max-nodes=${E2E_MAX_CLUSTER_NODES}"
```
Comment (chaodaiG) [inline scripts/e2e-tests.sh]: move `--min-node` and `max-nodes` out of "header" might be better

**Comment by chaodaiG** [comment]
overall looks good to me, just a formatting comment

File: scripts/e2e-tests.sh (hunk 1 above)
Comment (adrcunha) [reply scripts/e2e-tests.sh]: Agreed. Holding for the suggestion.

/hold

Comment (tcnghia) [reply scripts/e2e-tests.sh]: Done

Comment (chaodaiG) [reply scripts/e2e-tests.sh]: yeah that's exactly what I meant, something like this:
`echo "args:\n\t--min-nodes=${E2E_MIN_CLUSTER_NODES}\n\t--max-nodes=${E2E_MAX_CLUSTER_NODES}"`

Comment (adrcunha) [reply scripts/e2e-tests.sh]: I suggest a human-readable logging like

```echo "Cluster will have a minimum of ${E2E_MIN_CLUSTER_NODES} and a maximum of ${E2E_MIN_CLUSTER_NODES} nodes"```

Comment (chaodaiG) [reply scripts/e2e-tests.sh]: sounds good to me

Comment (tcnghia) [reply scripts/e2e-tests.sh]: Done

**Pull request merged**